iroh/src/discovery.rs

```diff
@@ -500,6 +499,7 @@ mod tests {
             self.shared
                 .nodes
                 .lock()
+                .unwrap()
```

Comment:
I would prefer if we used `.expect("poisoned")` as that way we can more easily treat `.unwrap()` as "un-audited".  (Having done this whole thing about auditing all our unwraps once before :wink: )

Reply:
fixed

Reply:
except this one?